plugins/user-activity-events/client.js

```diff
@@ -1,32 +1,10 @@
-var getPassive = function() {
-	var supported = false;
-	var options = Object.defineProperty({}, 'passive', {
-		// eslint-disable-next-line getter-return
-		get: function() { supported = true; }
-	});
-	try {
-		document.addEventListener('test', null, options);
-	} catch (e) { /**/ }
-
-	getPassive = supported
-		? function() { return { passive: true }; }
-		: function() { return false; };
-
-	return getPassive();
-};
```

Comment:
If I'm wrong here, please do correct me, but it seems like `passive` is a supported event listener option in all of our supported browsers (and even all of our maintenance browsers), so realistically I don't think we need this check.

Reply:
Yeah that's totally correct at this point -- good catch!